modules/core/src/main/java/org/apache/ignite/configuration/IgniteConfiguration.java

```diff
@@ -1032,6 +1039,29 @@ public IgniteConfiguration setSqlQueryHistorySize(int size) {
         return this;
     }
 
+    /**
+     * Defines the default query timeout.
+     *
+     * Defaults to {@link #DFLT_QRY_TIMEOUT}.
+     *
+     * @return Default query timeout.
+     */
+    public long getDfltQryTimeout() {
```

Comment:
I suppose according to our conventions a method name should be `getDefaultQueryTimeout`.